tools/mhm-tools/mhm_bankfull_discharge/bankfull_discharge.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="bankfull_discharge" name="Calculate Bankfull Discharge and Wetted Perimeter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>
+        Calculate the river discharge at bankfull conditions and the bankfull width.
+    </description>
+    <requirements>
+        <requirement type="package" version="3.8">python</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python3 $__tool_directory__/_cli/__init__.py bankfull
+        -i "${input}" 
+        -o "${output}"
+        -r "${return_period}" 
+        -v "${var}"
+        #if $wetted_perimeter:
+            --wetted_perimeter
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="input" type="data" format="netcdf" label="Input NetCDF file with discharge data" />
+        <param name="output" type="data" format="netcdf" label="Output NetCDF file" />
+        <param name="return_period" type="float" value="1.5" label="Return Period (years)" help="The return period of the flood in years." />
+        <param name="var" type="text" value="Qrouted" label="Variable name for routed streamflow in the NetCDF file" help="Name of the variable in the input file." />
```

Comment:
I would like to add a sanitizer / validator added to this text parameter. Otherwise users can input arbitrary text here (which is a security risk).